src/Rector/AbstractPHPUnitRector.php

```diff
@@ -2,21 +2,16 @@
 
 namespace Rector\Rector;
 
-use Nette\Utils\Strings;
 use PhpParser\Node;
+use PHPUnit\Framework\TestCase;
 use Rector\Node\Attribute;
 
 abstract class AbstractPHPUnitRector extends AbstractRector
 {
-    /**
-     * Check if the file is a PHPUnit TestCase. By default, it should end with "Test", as it is the standard.
-     *
-     * @see https://phpunit.de/getting-started-with-phpunit.html
-     */
     protected function isInTestClass(Node $node): bool
     {
-        $className = (string) $node->getAttribute(Attribute::CLASS_NAME);
+        $parentClassName = (string) $node->getAttribute(Attribute::PARENT_CLASS_NAME);
```

Comment:
@TomasVotruba Do you know any Attribute that indicates all class parents?

Reply:
All available `Attribute` items are in the `Attribute` class.

This should be resolved by TypeResolver, as it's used only sometimes. So I'd get `CLASS_NODE` and then get its types.

What do you think?

Reply:
@TomasVotruba Do you have an example of `TypeResolver` that I can study and update this PR?